### ISSUE TITLE
some references are hyperlinked (in addition to be listed at the bottom), some not

### ISSUE THREAD

**Issue by MalikaIhle** [lmu-osc/Simulations-for-Advanced-Power-Analyses#6]
perhaps this is because they are not all OA?

**Comment by nicebread** [comment]
No, this needs to be done. @MWiehr, can you look for open access version of the references and link them?

**Comment by MWiehr** [comment]
Should I prioritise looking for open access versions of the sources over looking at the other parts of the workshop?As I currently don't know how to embed the links: can I send you the OA information via mail? I think finding out how to do it otherwise might get time intensive.Should I do that for all workshops? Or just a particular one?LG Martin Am 16.03.2023 17:09 schrieb "Felix.S" ***@***.***>:
No, this needs to be done. @MWiehr, can you look for open access version of the references and link them?

**Comment by MalikaIhle** [comment]
[text](url)

**Comment by nicebread** [comment]
@MWiehr Please add a link to the OA version in each inline citation, and also at each reference in the reference list at the end; e.g. in LM1.qmd:

line 323: Button et al. (2015) --> Button et al. ([2015](https://doi.org/10.1017/S0033291715001270))

In reference list:

Button, K. S., Kounali, D., Thomas, L., Wiles, N. J., Peters, T. J., Welton, N. J., Ades, A. E., & Lewis, G. (2015). Minimal clinically important difference on the Beck Depression Inventory—II according to the patient’s perspective. Psychological Medicine, 45(15), 3269–3279. [https://doi.org/10.1017/S0033291715001270](https://doi.org/10.1017/S0033291715001270)

(this paper was easy, because it was open access already. In other cases you can use the https://openaccessbutton.org to find OA versions of published articles).

Yes, please prioritize this! (Gracia can do the reading of the other chapters)

**Comment by MWiehr** [comment]
As I have a driving lesson this morning, I will finish the links later today. As doing the hyperlinks seems doable I will also try to do that. If I run into any problems I will let you knowAm 17.03.2023 08:53 schrieb "Felix.S" ***@***.***>:
@MWiehr Please add a link to the OA version in each inline citation, and also at each reference in the reference list at the end; e.g. in LM1.qmd:
line 323: Button et al. (2015) --> Button et al. (2015)
In reference list:
Button, K. S., Kounali, D., Thomas, L., Wiles, N. J., Peters, T. J., Welton, N. J., Ades, A. E., & Lewis, G. (2015). Minimal clinically important difference on the Beck Depression Inventory—II according to the patient’s perspective. Psychological Medicine, 45(15), 3269–3279. https://doi.org/10.1017/S0033291715001270
(this paper was easy, because it was open access already. In other cases you can use the https://openaccessbutton.org to find OA versions of published articles).
Yes, please prioritize this! (Gracia can do the reading of the other chapters)

**Comment by nicebread** [comment]
Hi @MWiehr, OA-Versionen (also frei zugänglich, ohne Paywall) findet man am einfachsten auf (a) https://scholar.google.com oder auf (b) https://openaccessbutton.org.

Zu deinem ersten Link findet man es z.B: auf Scholar (siehe PDF Link rechts):
https://scholar.google.com/scholar?hl=de&as_sdt=0%2C5&q=Change+from+baseline+and+analysis+of+covariance+revisited&btnG=

Kannst du die fehlenden noch mal durchgehen? Danke!
